src/plugins/router.ts

```diff
@@ -138,6 +142,17 @@ const routes: RouteRecordRaw[] = [
   }
 ];
 
+/**
+ * TESTNET ONLY ROUTES
+ */
+if (isGoerli) {
```

Comment:
This is a computed property so needs .value
```suggestion
if (isGoerli.value) {
```